test/test_gipc.py

```diff
@@ -271,7 +270,8 @@ def test_pid(self):
         _call_close_method_if_exists(p)
 
     def test_terminate(self):
-        p = start_process(gevent.sleep, args=(1,))
+        # 1s is too long, and the process is killed on python 3.8.6
+        p = start_process(gevent.sleep, args=(0.5,))
```

Comment:
This is an interesting change -- not sure if I really understand it.

Reply:
With the longer sleep, I was periodically getting a SIGKILL instead of a SIGTERM, and was unsure why. Decreasing the timeout made SIGTERM come through more stable.

It was an earlier change, so I can test if 1s works stable again, and re-increase it... (technically, 100 should work too)

Reply:
I reverted this, and also tested with longer values... and it appears not to suffer the same instability it was before. So I think we're ok